docs/docs/k8s/ingress.md

```diff
@@ -244,6 +244,9 @@ Additional TLS certificates may be supplied by creating a Kubernetes secret(s) i
 
 Please note that the referenced `tls_client_secret` must be a [TLS Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets). `tls_custom_ca_secret` and `tls_downstream_client_ca_secret` must contain `ca.crt` containing a .PEM encoded (base64-encoded DER format) public certificate.
 
+::: tip
+If you're using a wildcard certificate on all routes using the Pomerium IngressClass, you can set the secret for the cert globally when defining the class. See the [Pomerium Ingress Controller README](https://github.com/pomerium/ingress-controller#ingressclass) for more information.
```

Comment:
why refer to github, why not put it here for consistency.

Reply:
Because, as you noted before, we don't document defining the IngressClass anywhere else in docs since it's created by the Helm chart, and our docs are based around an installation with Helm.
In other words, documenting this annotation in the docs would also mean creating a new section, possibly a new page, on the IngressController class.

With that said, does this decision make sense, or should we discuss implementing such a section/page?

Reply:
I believe we should document Helm option that sets this annotation for a chart. 
However it also makes sense to describe this annotation in the IngressClass separate section.

Reply:
Moved to a different doc in a different form.